modules/roles_profiles/manifests/profiles/gecko_1_b_osx_1015_generic_worker.pp

```diff
@@ -0,0 +1,160 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::gecko_1_b_osx_1015_generic_worker (
+    String $worker_type = 'gecko-1-b-osx-1015',
```

Comment:
The `$worker_type` seems unique to this profile anyway.  Let's just move this down a couple lines and remove it from the parameters.

Reply:
np, this isn't necessary